static/app/utils/replays/types.tsx

```diff
@@ -57,8 +57,8 @@ type ExtraBreadcrumbTypes =
   | {
       category: 'navigation';
       data: {
-        from: string;
         to: string;
+        from?: string;
```

Comment:
marked as optional because the SDK currently isn't sending a field for `from` but we can reuse this crumb type

Reply:
That could be a good comment on top of the prop

Reply:
i'll pose it a a question in the channel